frontend/src/pages/destructionlist/detail/components/DestructionListEdit/DestructionListEdit.tsx

```diff
@@ -135,6 +135,21 @@ export function DestructionListEdit() {
     undefined,
   );
 
+  if (dataGridProps.fields && destructionList.processingStatus !== "new") {
+    dataGridProps.fields.splice(1, 0, {
```

Comment:
I personally prefer non-mutable approaches like spreading (e.g. `[...datagridProps.fields, processingStatus]`) but I'll leave this up to you. 

Can we move the status column to the right, I think the design stats this (in another context) as well.